### DISCUSSION TITLE
Error during new data set learning process

### DISCUSSION THREAD

**Discussion by yerin723** [mindee/doctr#1293] [General]
hello.
Currently, we created a detection data set through python labelImg.py.
https://github.com/mindee/doctr/tree/main/references/detection
I also created a data format structure by referring to the address above.

But in the terminal window
python references/detection/train_pytorch.py ​​C:\Users\PIAI\Desktop\test C:\Users\PIAI\Desktop\val db_resnet50 --epochs 5 --device NVIDIA GeForce RTX 2080

When I type it like this
ImportError: cannot import name 'VOCABS' from partially initialized module 'doctr.datasets' (most likely due to a circular import) (C:\Users\PIAI\Desktop\ocrproject\doctr\doctr\datasets\__init__.py)

This error keeps occurring.

What should I do to resolve the error?
I am a beginner, so please explain it step by step.

**Comment by felixT2K** [comment]
Hi @yerin723 👋🏼 

1. Fork the repo and clone it 
2. `cd doctr` -> `pip3 install -e .[torch]`
3. `pip3 install fastprogress wandb psutil`
4. `python3  references/detection/train_pytorch.py <path/to/your/train_set> <path/to/your/val_set> db_resnet50 --epochs 5 --device 0`
